readme.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
```

Comment:
Do you have a better name for this file? Any particular directory it should go into?